Web Page Clone of Newsweek we page

abdelp opened the pull request:
Responsive design of Newsweek webpage

Based on the Odin Project Requirements specified here: https://www.theodinproject.com/courses/html5-and-css3/lessons/using-bootstrap

use of grids with bootstrap framework v4.4.1

abdelp committed:
Add initial html structure

abdelp committed:
Add initial html structure

abdelp committed:
Add header for xl devices

abdelp committed:
Add initial header-nav structure

abdelp committed:
Add aside section in the main part and the top-story section

abdelp committed:
Add latest news section structure

abdelp committed:
Add more-stories section structure

abdelp committed:
Add signup block

abdelp committed:
Add initial in-the-mazine section structure

abdelp committed:
Add initial editor's picks section structure

abdelp committed:
Add u.s, world, business, tech & science, culture, sport, and
subscriptions sections initial structure

abdelp committed:
Add initial footer structure

abdelp committed:
fixed top-header styling (finished)

abdelp committed:
fixed top-header for all devices in chrome inspector

abdelp committed:
Fix bottom-header section for all devices

abdelp committed:
Fix features stories styling and responsivness and order of top story in
resizing

abdelp committed:
modified styling and structure for top story section

abdelp committed:
Update opinion section styling

abdelp committed:
Update culture & travel section styling

abdelp committed:
Update latest-news section styling

abdelp committed:
Update signup block styling

abdelp committed:
Update the footer structure and styling for all devices

abdelp committed:
Update subscription section styling for all devices

abdelp committed:
Update subscribe section title margin

abdelp committed:
Updated min-height and height of newsletter image

abdelp committed:
Update topics-section for all devices

abdelp committed:
Update featured-slideshows section styling and structure for all devices

abdelp committed:
Fix all stylint error on style.css file

abdelp committed:
Fix all html errors

abdelp committed:
Update sections in html file

abdelp committed:
Update README.md file

pelzolga123 commented:
### Hi Abdel! I am marking this review request as invalid, because your linter is not setup for this repository. 
Please kindly setup linter and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

abdelp committed:
Update .gitingore file

abdelp commented:
Enabled linter in the repository

addod19 commented:
## Pull Request under review by Daniel Larbi Addo (TSE)

addod19 commented on assets/stylesheets/style.css:
```diff
@@ -0,0 +1,580 @@
+  background-color: #f4f5f6;
+}
+
+.page-header,
+.main,
+.footer {
```
I suggest you give these classes a ```width:100%```. To solve the horizontal scroll we get on tablet and mobile screen sizes

addod19 reviewed (changes requested):
## Status: Require changes 🔴 

Hi @abdelp, you did a great job enabling linter for your repo. 👍 However, there are some few recommendations. Find comment below:
- Kindly fix the warning
![sl](https://user-images.githubusercontent.com/46388113/74065800-968b2680-49ed-11ea-9f24-ac67c2c9ea1d.PNG)
 returned by [HTML markup](https://validator.w3.org/)
- ```Tablet mode``` Currently this is how it looks **(1 column)**
![new1](https://user-images.githubusercontent.com/46388113/74066127-6abc7080-49ee-11ea-8208-1908d2147e70.PNG) It should be like this **(2 columns)**
![onew1](https://user-images.githubusercontent.com/46388113/74066167-80319a80-49ee-11ea-81df-e6a29f3c319a.PNG)
- Currently,  this part looks like this
![not](https://user-images.githubusercontent.com/46388113/74066408-efa78a00-49ee-11ea-9148-4007da757c14.PNG)
It should be this
![an](https://user-images.githubusercontent.com/46388113/74066459-077f0e00-49ef-11ea-8d7a-be613c3a433d.PNG)
- Kindly make the following changes and aligns on breakpoints:
Featured Stories, Top story, Opinion, Latest news, more stories, In the magazine, editors pick, Featured slidshows
![co](https://user-images.githubusercontent.com/46388113/74067130-6bee9d00-49f0-11ea-9d19-6c605e765aab.PNG)
- There is a horizontal scroll. 

Kindly make these changes and submit another code review 🙂 
Happy coding 💻 
Daniel Larbi Addo

abdelp committed:
Fixed aligment and order of top-stories

abdelp committed:
Fix horizontal scroll on header, updated resizing of elements

abdelp committed:
Remove responsive reporter file

abdelp committed:
Fix error on css file

abdelp commented:
Removed warning from the linter
Updated the alignment for medium sizes devices on top history
fixed the horizontal scroll
Updated alignment for sections

abdelp committed:
Update live demo link on README.md file

abdelp committed:
Update app_screenshot.png file

Anna-Myzukina commented:
### Pull Request under the review by Anna Muzykina (TSE)

Anna-Myzukina commented:
## Hi @abdelp !
### Nice job! Project looks! :+1: 

### But your project needs a few changes before it can be approved.
#### Implementation details


*  It adds [viewport metatag](https://getbootstrap.com/docs/4.3/getting-started/introduction/#responsive-meta-tag) 
![screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/using_bootstrap/bootstrap_requirement.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/using-bootstrap)
*  It uses Bootstrap Css classes **everywhere** for styling (for example [Grid layout](https://getbootstrap.com/docs/4.3/layout/overview/) and [Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options)) 
![screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/using_bootstrap/bootstrap_classes_requirement.png) from [Pathwrigth](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540308/)

### Main content

**It changes with RWD breakpoints** ([link to page](https://www.newsweek.com/))

* Kindly recommended checking all parts of your project, it still has a big difference:

#### Your version
![us1](https://user-images.githubusercontent.com/26605804/74072548-d06d2480-4a0f-11ea-990c-3bc97cc67728.png)
#### Original
![us](https://user-images.githubusercontent.com/26605804/74072538-ccd99d80-4a0f-11ea-8bcb-0e4efa6fd89f.png)

#### Your version
![subscr](https://user-images.githubusercontent.com/26605804/74072552-d400ab80-4a0f-11ea-8eac-3cd953292970.png)

#### Original
![subs](https://user-images.githubusercontent.com/26605804/74072566-de22aa00-4a0f-11ea-8ace-f52bd77d96e3.png)
![newsl](https://user-images.githubusercontent.com/26605804/74072578-e24ec780-4a0f-11ea-8fda-cd925935a9f1.png)


*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding *

Anna-Myzukina reviewed (changes requested):
The project still needs changes

abdelp committed:
Update breakpoints

abdelp committed:
Update breakpoints in topics section for all elements

abdelp committed:
Update background-bottom classes

abdelp committed:
Update margin, padding text-color, text-transform with bootstrap classes

abdelp committed:
Update boostrap classes of padding

abdelp committed:
Update boostrap classes of padding

abdelp committed:
Update classes to bootstrap

abdelp committed:
fix horizontal for the header-nav

abdelp committed:
Fix horizontal scroll by removing all unnecesary container element
classes

abdelp committed:
Remove commented code from index.html and style.css and validate both

abdelp committed:
Remove media query reporter css file

abdelp committed:
Aligned text in subscription section

abdelp committed:
Update live demo link

nikasulo reviewed (approved):
# Your hard work has paid off! :rocket: 
This is really great work @abdelp :handshake: Please merge this and be on your way :smile:. See you at the top :medal_military:  

#### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

abdelp commented:
Thank you very much, guys!
